importer/main.py

```diff
@@ -861,8 +862,75 @@ def clean_duplicates(users, cascade_delete):
                 logging.info("No Practitioners found")
 
 
+# Create a csv file and initialize the CSV writer
+def write_csv(data, resource_type, fieldnames):
+    logging.info("Writing to csv file")
+    current_time = datetime.now().strftime("%Y-%m-%d-%H-%M")
+    csv_file = f"csv/{current_time}-export_{resource_type}.csv"
+    with open(csv_file, 'w', newline='') as file:
+        csv_writer = csv.writer(file)
+        csv_writer.writerow(fieldnames)
+        csv_writer.writerows(data)
+
+
+# This function exports resources from the API to a csv file
+def export_resources_to_csv(resource_type, parameter, value, batch_size):
+    resource_type = get_valid_resource_type(resource_type)
+    resource_url = "/".join([config.fhir_base_url, resource_type])
+    if len(parameter) > 0:
+        resource_url = resource_url + "?" + parameter + "=" + value + "&_count=" + str(batch_size)
+    response = handle_request("GET", "", resource_url)
+    if response[1] == 200:
+        resources = json.loads(response[0])
+        data = []
+        if resource_type == "Location":
+            elements = ["name", "status", "id", "identifier", "parentName", "parentID", "type", "typeCode",
+                        "physicalType", "physicalTypeCode"]
+        elif resource_type == "Organization":
+            elements = ["name", "active", "id", "identifier", "alias"]
+        elif resource_type == "CareTeam":
+            elements = ["name", "status", "id", "identifier", " organizations", "participants"]
+        else:
+            elements = []
+        for x in resources["entry"]:
```

Comment:
we need to handle a case where the response returned has 0 resources and thus does not have "entry"